### PR TITLE
[core] Fix deprecated class

### PR DESCRIPTION
**Pull request by VSevagen**
django_mysql.models.JSONField is deprecated. I replaced it
with the equivalent provided through the log

Signed-off-by: sevagenv <sevagenv@gmail.com>

### CODE REVIEW

**Comment by VSevagen** [comment]
@sduenas It seems like django.db.models.JSONField is not available in Django 2.1. It was added from django 3.1. Also, in the requirements.txt file, there is a mismatch with django-mysql and django version. django-mysql does not support django2.1. django should be >=2.2
Should I bump django to 3.1 or just leave it as it is ? i think it is the current django version that is causing an issue.

**Comment by sduenas** [comment]
I won't have any problem to upgrade the code to Django 3.1. It would be better. My only concern is that the related libraries such as `graphene-django`, `django-rq`, etc, are compatible with this version of Django. If they are, go ahead and update the PR.

**Comment by VSevagen** [comment]
I updated the requirements. Only `mysqlclient` has issue with django, updated that and for `django-mysql`, the latest version is currently 3.11.1

**Comment by VSevagen** [comment]
The exceptions are being raised but the error messages do not match. The easy fix wouldd be to modify `INVALID_BOOLEAN_CHECK_ERROR` but I doubt this is the proper way of fixing this issue.

**Comment by VSevagen** [comment]
> The exceptions are being raised but the error messages do not match. The easy fix wouldd be to modify `INVALID_BOOLEAN_CHECK_ERROR` but I doubt this is the proper way of fixing this issue.

I checked whether any dependency was causing this issue. Turns out its Django. Keeping all the other deps changes as they are, the issue arises when I transition from django2.2 to >=django3.0. It seems that the validation error message is shown in a list or dict format from django 3.0 onwards.

**Review by sduenas — CHANGES_REQUESTED** [review]
LGTM but I think we should remove the dependency with `django-mysql`.

File: requirements.txt [hunk 1]
```diff
@@ -1,10 +1,10 @@
 click==7.1.1
-Django==2.1
-django-mysql>=3.2.0
+Django==3.1
+django-mysql>=3.11.1
```
[the remark this answers is not in the document]
Comment (VSevagen) [reply requirements.txt]: I removed it in pyproject.toml as well.

**Review by sduenas — CHANGES_REQUESTED** [review]
Thanks for your changes but I forgot to mention to update the README file. It has references to Django version and this django-mysql package.

**Review by sduenas — APPROVED** [review]
LGTM

**Comment by sduenas** [comment]
@VSevagen I saw there's some problems with your branch. It's not rebased propertly. Would you mind to fix it?

**Comment by VSevagen** [comment]
@sduenas I rebased it.

**Review by sduenas — APPROVED** [review]
LGTM

**Comment by sduenas** [comment]
Thanks for your contribution @VSevagen .

**Pull request merged**